diff --git a/system/configuration.nix b/system/configuration.nix
index 1e60982..29e58e3 100644
--- a/system/configuration.nix
+++ b/system/configuration.nix
@@ -183,14 +183,6 @@ in
     };
   };
 
-  # Configure keymap in X11
-  services.xserver = {
-      layout = "se";
-      xkbVariant = "";
-      xkbModel = "pc105";
-      xkbOptions = "eurosign:e, compose:menu, grp:caps_toggle";
-  };
-
   # Configure console keymap
 
   i18n.extraLocaleSettings = {
diff --git a/system/wm/xmonad.nix b/system/wm/xmonad.nix
index a61b204..f1fa41d 100644
--- a/system/wm/xmonad.nix
+++ b/system/wm/xmonad.nix
@@ -14,7 +14,7 @@
 
     xserver = {
       enable = true;
-      autorun = true; ## added 20 aug 2022 /Ogg
+      # autorun = true; ## added 20 aug 2022 /Ogg
 
       # extraLayouts.us-custom = {
       #   description = "US layout with custom hyper keys";
@@ -22,7 +22,11 @@
       #   symbolsFile = ./us-custom.xkb;
       # };
 
-      layout = "se";
+      # Configure keymap in X11
+      layout = "us";
+      /* xkbVariant = "";
+      xkbModel = "pc105";
+      xkbOptions = "eurosign:e, compose:menu, grp:caps_toggle"; */
 
       libinput = {
         enable = true;
@@ -37,9 +41,8 @@
 
       displayManager = {
         defaultSession = "none+xmonad";
-        lightdm.enable = false; ## added 20 aug 2022 /Ogg
-        startx.enable = true;   ## added 20 aug 2022 /Ogg
-
+        /* lightdm.enable = false; ## added 20 aug 2022 /Ogg
+        startx.enable = true; ## added 20 aug 2022 /Ogg */
       };
 
       windowManager.xmonad = {
@@ -60,7 +63,7 @@
       General = {
         Enable = "Source,Sink,Media,Socket";
       };
-   };
+    };
   };
 
   services.blueman.enable = true;
